.vscode/settings.json

```diff
@@ -0,0 +1,25 @@
+{
+  "search.exclude": {
+    ".storybook/data": true,
+    "assets/vendor": true,
```

Comment:
We don't have the `vendor` directory in the `assets` folder. Let's remove this line. Plus let's add `vendor/`, `third-party/` and `node_modules/` paths to exclude from the search.

Reply:
Whoops, indeed! I think that was part of an older config maybe? Part of this config is what's been on my machine a long time 😅 

Thanks, changed!